src/types/plugins.ts

```diff
@@ -97,31 +97,31 @@ export type ClientPluginManifest = {
     };
 }
 
-export type MarketplaceLabel = {
+export type MarketplaceLabel = { // TODO remove this in favour of the definition in types/marketplace after the mattermost-redux migration
```

Comment:
I was going to re-export the types from `types/marketplace`, but I was worried about type import loops causing some weird problems, so I figured we could just duplicate the types for the time being.